app/src/main/java/umc/everyones/lck/presentation/mypage/MyPageMyteamFragment.kt

```diff
@@ -0,0 +1,138 @@
+package umc.everyones.lck.presentation.mypage
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.animation.Animation
+import android.view.animation.AnimationUtils
+import android.widget.ImageView
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.lifecycleScope
+import androidx.navigation.fragment.findNavController
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.launch
+import umc.everyones.lck.R
+import umc.everyones.lck.databinding.DialogMyteamConfirmBinding
+import umc.everyones.lck.databinding.FragmentMypageMyteamBinding
+import umc.everyones.lck.presentation.base.BaseFragment
+import umc.everyones.lck.util.TeamData
+
+@AndroidEntryPoint
+class MyPageMyteamFragment : BaseFragment<FragmentMypageMyteamBinding>(R.layout.fragment_mypage_myteam) {
+
+    private var selectedTeamName: String? = null
+    private var isScrollViewExpanded = false // Track the state of the ScrollView
+
+    override fun initObserver() {
+        // ViewModel 관찰이 필요한 경우 추가
+    }
+
+    override fun initView() {
+        setupTeamSelection { teamName ->
+            selectedTeamName = teamName
+            Log.d("MyPageMyteamFragment", "Selected team: $selectedTeamName")
+        }
+
+        binding.tvMypageMyteamTopbarEdit.setOnClickListener {
+            if (selectedTeamName == null) {
+                showTeamConfirmDialog()
+            } else {
+                lifecycleScope.launch {
+                    try {
+                        Log.d("MyPageMyteamFragment", "Navigated to next fragment with team: $selectedTeamName")
+                    } catch (e: Exception) {
+                        Log.e("MyPageMyteamFragment", "Error navigating", e)
+                    }
+                }
+            }
+        }
+
+        // Initialize the ScrollView and Arrow Button
+        binding.layoutMypageMyteamList.visibility = View.GONE
+        binding.btnMypageMyteamArrow.setImageResource(R.drawable.ic_arrow_down)
+
+        // Set up click listener for the arrow button
+        binding.btnMypageMyteamArrow.setOnClickListener {
+            toggleScrollView()
+        }
+
+        binding.ivMypageMyteamBack.setOnClickListener{
+            requireActivity().onBackPressed()
```

Comment:
```suggestion
            findNavController().navigateUp()
``` 
navigation에서 뒤로가기는 다음과 같이 하시면 됩니당